usl_pipeline/usl_pipeline/readers/elevation.py

```diff
@@ -0,0 +1,54 @@
+import typing
+
+import rasterio.features
+
+from usl_pipeline.shared.entities import Elevation
+from usl_pipeline.shared.entities import ElevationHeader
+
+
+def read_from_geotiff(
+    file: typing.BinaryIO,
+    header_only: bool = False,
+    band: int = 1,
+    no_data_value: float = None,
```

Comment:
ah, likewise, the type here is `float | None` or `Optional[None]`

Reply:
Done